src/main/java/org/brapi/test/BrAPITestServer/service/core/ListService.java

```diff
@@ -273,9 +273,12 @@ private void updateEntity(ListEntity entity, @Valid ListNewRequest list) throws
 				item.setList(null);
```

Comment:
Is this still needed? It seems like the entity.getData() list will be cleared in both cases below. Are the item references used for anything?

Reply:
It's not needed and is gone, now.